cmd/antrea-agent/agent.go

```diff
@@ -150,6 +152,7 @@ func run(o *Options) error {
 
 	go networkPolicyController.Run(stopCh)
 
+	healthzCheckers := make([]healthz.HealthzChecker, 0)
```

Comment:
Sorry, I might be not clear, I meant currently there is no point to put dummy healthcheck to agentMonitor. We can consider turning some check in agent monitor to apiserver's healthcheck in future but not the scope of this PR, so you can restore the agentmonitor code.
I think you need to add probes to antrea.yaml, otherwise this cannot resolve the original issue.